lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -140,6 +140,7 @@
 // ====================
 .instructor-dashboard-content-2 {
   @extend .content;
+  display: block; // override the table-cell display set in .content
```

Comment:
Should we not update ```.content``` to not default to ```table-cell```? If not for this fix in a future ticket?